modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/validators/ValidationService.java

```diff
@@ -1,6 +1,6 @@
 /********************************************************************************
- * Copyright (c) 2022, 2023 T-Systems International GmbH
- * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022, 2023, 2024 T-Systems International GmbH
+ * Copyright (c) 2022, 2023, 2024 Contributors to the Eclipse Foundation
```

Comment:
Look at [Copyright Headers](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) of the Handbook. There is no need to include all years, you may include only the year in which the content was created. Alternatively, you can keep year when a file was updated. Intermediate years are not needed

Reply:
Good point @dvasunin! Although I have to emphasise the you have to use either one year which is when the file was created (this is the one you mentioned) or 2 years where the first is the creation year and second is the last modification year.

Reply:
Updated as per Handbook